docs/configure-kibana-endpoint.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[setup-kibana-endpoint]]
+== Configure the Kibana endpoint
+
+Starting with {beatname_uc} 7.3.0, we've introduced a new Kibana API for APM Server.
+Configuring this endpoint is required for 
+{kib-ref}/agent-configuration.html[APM Agent configuration in Kibana].
+Once the endpoint is configured,
+you'll be able to fine-tune certain agent configurations directly in Kibana without having to redeploy your agents.
```

Comment:
Approved already, but forgot to mention!!: somewhere over here could note that the API between agent and server for cofig is secured by the same secret_token used for other endpoints, if it is configured.

Also forgotten that we have a whole API section (`intake-api`) for which we have a new `/config/v1/agents` entry that could be named Agent Config API, i guess?